docs/iot/getting-started/tizen-image-download-flash.md

```diff
@@ -152,7 +152,7 @@ The IoT Setup Manager view contains the following fields:
 
 -   **Select Drive**
```

Comment:
Please change the following:
Line 59 – Please change “Windows 7 and above (64 and 32 bit)” to “Windows 7 and earlier (64-bit and 32-bit)”
Line 60 – Please change “Ubuntu 14.04 LTS and above (64 and 32 bit)” to “Ubuntu 14.04 LTS and earlier (64-bit and 32-bit)”
Line 64 – Please change “Java Runtime Environment (JRE) version 1.8 or above” to “Java Runtime Environment (JRE) version 1.8 or earlier”

Reply:
I think it should be “Windows 7 and later (64 and 32 bit)”, “Ubuntu 14.04 LTS and later (64-bit and 32-bit)” and  “Java Runtime Environment (JRE) version 1.8 or later”

Reply:
@jpark0123, it is correct to use later.